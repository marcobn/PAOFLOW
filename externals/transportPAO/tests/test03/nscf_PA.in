&CONTROL
    prefix='poly_pa'
    calculation='nscf',
    restart_mode='from_scratch',
    pseudo_dir='../pseudo/',
    outdir='./SCRATCH',
    wf_collect=.TRUE.
/
&SYSTEM
        ibrav = 8,
     celldm(1)=30, celldm(2) = 1.0000,
    celldm(3) = 0.155566,
          nat = 4,
         ntyp = 2,
      ecutwfc = 40.0 ,
         nbnd = 30
        nosym = .TRUE.
        noinv = .TRUE.
    input_dft = 'PZ'
/
&ELECTRONS
    electron_maxstep = 50,
    mixing_mode = 'plain',
    mixing_beta = 0.3,
    conv_thr =  1.0d-8,
/
ATOMIC_SPECIES                                       
    C    1.00000  C.pw-mt_fhi.UPF
    H    1.00000  H.pz-vbc.UPF
ATOMIC_POSITIONS bohr
C      0.000000000     0.000000000    -1.106970000
C      1.225187328     0.000000000     1.106976303
H     -2.035217982     0.000000000    -1.115497550
H      3.260405310     0.000000000     1.115503853
K_POINTS automatic
1 1 30 0 0 0

