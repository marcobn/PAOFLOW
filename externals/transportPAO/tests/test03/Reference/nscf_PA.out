
     Program PWSCF v.5.0.2 (svn rev. 9392) starts on 17Apr2014 at 20:37:34 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote.php

     Parallel version (MPI), running on     6 processors
     R & G space division:  proc/nbgrp/npool/nimage =       6

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  3
     Reading input from /home/ferretti/want/tests/test16/nscf_PA.in

     IMPORTANT: XC functional enforced from input :
     Exchange-correlation      = PZ ( 1 1 0 0 0)
     EXX-fraction              =        0.00
     Any further DFT definition will be discarded
     Please, verify this is what you really want

               file C.pw-mt_fhi.UPF: wavefunction(s)  4f renormalized
               file H.pz-vbc.UPF: wavefunction(s)  1S renormalized

     Atomic positions and unit cell read from directory:
     ./SCRATCH/poly_pa.save/
 

     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used

 
     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min        1907    1907    584                23948    23948    4049
     Max        1909    1909    586                23949    23949    4050
     Sum       11449   11449   3513               143691   143691   24299
 


     bravais-lattice index     =            8
     lattice parameter (alat)  =      30.0000  a.u.
     unit-cell volume          =    4200.2820 (a.u.)^3
     number of atoms/cell      =            4
     number of atomic types    =            2
     number of electrons       =        10.00
     number of Kohn-Sham states=           30
     kinetic-energy cutoff     =      40.0000  Ry
     charge density cutoff     =     160.0000  Ry
     Exchange-correlation      = PZ ( 1 1 0 0 0)
     EXX-fraction              =        0.00

     celldm(1)=  30.000000  celldm(2)=   1.000000  celldm(3)=   0.155566
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (   1.000000   0.000000   0.000000 )  
               a(2) = (   0.000000   1.000000   0.000000 )  
               a(3) = (   0.000000   0.000000   0.155566 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  1.000000  0.000000  0.000000 )  
               b(2) = (  0.000000  1.000000  0.000000 )  
               b(3) = (  0.000000  0.000000  6.428140 )  


     PseudoPot. # 1 for  C read from file:
     ../pseudo/C.pw-mt_fhi.UPF
     MD5 check sum: 13ff76dfb29e90984447383659675a25
     Pseudo is Norm-conserving, Zval =  4.0
     Generated using FHI98PP, converted with fhi2upf.x v.5.0.1
     Using radial grid of  461 points,  3 beta functions with: 
                l(1) =   0
                l(2) =   1
                l(3) =   3

     PseudoPot. # 2 for H  read from file:
     ../pseudo/H.pz-vbc.UPF
     MD5 check sum: 90becb985b714f09656c73597998d266
     Pseudo is Norm-conserving, Zval =  1.0
     Generated by new atomic code, or converted to UPF format
     Using radial grid of  131 points,  0 beta functions with: 

     atomic species   valence    mass     pseudopotential
        C              4.00     1.00000      C( 1.00)
        H              1.00     1.00000     H ( 1.00)

     No symmetry found



   Cartesian axes

     site n.     atom                  positions (alat units)
         1           C   tau(   1) = (   0.0000000   0.0000000  -0.0368990  )
         2           C   tau(   2) = (   0.0408396   0.0000000   0.0368992  )
         3           H   tau(   3) = (  -0.0678406   0.0000000  -0.0371833  )
         4           H   tau(   4) = (   0.1086802   0.0000000   0.0371835  )

     number of k points=    30
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   0.0666667
        k(    2) = (   0.0000000   0.0000000   0.2142713), wk =   0.0666667
        k(    3) = (   0.0000000   0.0000000   0.4285427), wk =   0.0666667
        k(    4) = (   0.0000000   0.0000000   0.6428140), wk =   0.0666667
        k(    5) = (   0.0000000   0.0000000   0.8570853), wk =   0.0666667
        k(    6) = (   0.0000000   0.0000000   1.0713566), wk =   0.0666667
        k(    7) = (   0.0000000   0.0000000   1.2856280), wk =   0.0666667
        k(    8) = (   0.0000000   0.0000000   1.4998993), wk =   0.0666667
        k(    9) = (   0.0000000   0.0000000   1.7141706), wk =   0.0666667
        k(   10) = (   0.0000000   0.0000000   1.9284419), wk =   0.0666667
        k(   11) = (   0.0000000   0.0000000   2.1427133), wk =   0.0666667
        k(   12) = (   0.0000000   0.0000000   2.3569846), wk =   0.0666667
        k(   13) = (   0.0000000   0.0000000   2.5712559), wk =   0.0666667
        k(   14) = (   0.0000000   0.0000000   2.7855273), wk =   0.0666667
        k(   15) = (   0.0000000   0.0000000   2.9997986), wk =   0.0666667
        k(   16) = (   0.0000000   0.0000000  -3.2140699), wk =   0.0666667
        k(   17) = (   0.0000000   0.0000000  -2.9997986), wk =   0.0666667
        k(   18) = (   0.0000000   0.0000000  -2.7855273), wk =   0.0666667
        k(   19) = (   0.0000000   0.0000000  -2.5712559), wk =   0.0666667
        k(   20) = (   0.0000000   0.0000000  -2.3569846), wk =   0.0666667
        k(   21) = (   0.0000000   0.0000000  -2.1427133), wk =   0.0666667
        k(   22) = (   0.0000000   0.0000000  -1.9284419), wk =   0.0666667
        k(   23) = (   0.0000000   0.0000000  -1.7141706), wk =   0.0666667
        k(   24) = (   0.0000000   0.0000000  -1.4998993), wk =   0.0666667
        k(   25) = (   0.0000000   0.0000000  -1.2856280), wk =   0.0666667
        k(   26) = (   0.0000000   0.0000000  -1.0713566), wk =   0.0666667
        k(   27) = (   0.0000000   0.0000000  -0.8570853), wk =   0.0666667
        k(   28) = (   0.0000000   0.0000000  -0.6428140), wk =   0.0666667
        k(   29) = (   0.0000000   0.0000000  -0.4285427), wk =   0.0666667
        k(   30) = (   0.0000000   0.0000000  -0.2142713), wk =   0.0666667

     Dense  grid:   143691 G-vectors     FFT dimensions: ( 125, 125,  20)

     Largest allocated arrays     est. size (Mb)     dimensions
        Kohn-Sham Wavefunctions         1.38 Mb     (   3005,   30)
        NL pseudopotentials             1.01 Mb     (   3005,   22)
        Each V/rho on FFT grid          0.95 Mb     (  62500)
        Each G-vector array             0.18 Mb     (  23949)
        G-vector shells                 0.05 Mb     (   6184)
     Largest temporary arrays     est. size (Mb)     dimensions
        Auxiliary wavefunctions         5.50 Mb     (   3005,  120)
        Each subspace H/S matrix        0.22 Mb     ( 120, 120)
        Each <psi_i|beta_j> matrix      0.01 Mb     (     22,   30)

     The potential is recalculated from file :
     ./SCRATCH/poly_pa.save/charge-density.dat

     Starting wfc are   34 randomized atomic wfcs

     total cpu time spent up to now is        0.4 secs

     per-process dynamical memory:    21.6 Mb

     Band Structure Calculation
     Davidson diagonalization with overlap

     ethr =  1.00E-10,  avg # of iterations = 39.3

     total cpu time spent up to now is      176.6 secs

     End of band structure calculation

          k = 0.0000 0.0000 0.0000     band energies (ev):

   -20.4724 -13.9107  -8.9230  -8.8675  -7.9626  -0.2392  -0.0875   0.2876
     0.5537   0.6398   0.6977   1.0973   1.2457   1.4144   1.5621   1.7443
     2.0169   2.3108   2.6717   2.6984   2.9972   2.9991   3.0872   3.0982
     3.1365   3.2909   3.5555   4.1707   4.3892   4.7203

          k = 0.0000 0.0000 0.2143     band energies (ev):

   -20.4540 -13.8937  -8.9732  -8.8441  -7.9902  -0.2120  -0.0640   0.3047
     0.5809   0.6658   0.7241   1.1249   1.2729   1.4408   1.5845   1.7654
     2.0457   2.3381   2.6983   2.7265   3.0234   3.0262   3.1146   3.1257
     3.1628   3.3094   3.5791   4.2099   4.4045   4.7476

          k = 0.0000 0.0000 0.4285     band energies (ev):

   -20.3985 -13.8426  -9.1347  -8.7716  -8.0565  -0.1309   0.0062   0.3535
     0.6627   0.7434   0.8034   1.2074   1.3548   1.5200   1.6510   1.8290
     2.1311   2.4199   2.7782   2.8103   3.1019   3.1077   3.1968   3.2080
     3.2409   3.3643   3.6491   4.3119   4.4561   4.8295

          k = 0.0000 0.0000 0.6428     band energies (ev):

   -20.3062 -13.7586  -9.4183  -8.6512  -8.1309   0.0029   0.1216   0.4275
     0.7988   0.8715   0.9361   1.3447   1.4912   1.6511   1.7601   1.9352
     2.2710   2.5562   2.9110   2.9491   3.2324   3.2432   3.3336   3.3450
     3.3642   3.4584   3.7637   4.4359   4.5680   4.9660

          k = 0.0000 0.0000 0.8571     band energies (ev):

   -20.1774 -13.6432  -9.8187  -8.4840  -8.1998   0.1876   0.2793   0.5130
     0.9894   1.0487   1.1228   1.5363   1.6821   1.8325   1.9098   2.0844
     2.4637   2.7470   3.0962   3.1421   3.4140   3.4315   3.5058   3.5260
     3.5366   3.6135   3.9188   4.5576   4.7584   5.1566

          k = 0.0000 0.0000 1.0714     band energies (ev):

   -20.0121 -13.5004 -10.3024  -8.2696  -8.2653   0.4203   0.4765   0.5846
     1.2343   1.2728   1.3644   1.7821   1.9274   2.0604   2.0978   2.2791
     2.7083   2.9922   3.3327   3.3889   3.6317   3.6449   3.6865   3.7721
     3.7828   3.8461   4.1092   4.7067   5.0080   5.3382

          k = 0.0000 0.0000 1.2856     band energies (ev):

   -19.8108 -13.3361 -10.8381  -8.3407  -8.0091   0.6032   0.6786   0.7264
     1.5336   1.5426   1.6610   2.0820   2.2267   2.3253   2.3273   2.5276
     3.0046   3.2917   3.6183   3.6893   3.7854   3.9225   3.9807   4.0734
     4.0835   4.1426   4.3336   4.9126   5.3063   5.5053

          k = 0.0000 0.0000 1.4999     band energies (ev):

   -19.5737 -13.1679 -11.3839  -8.4347  -7.7017   0.5202   0.9257   1.0549
     1.8578   1.8871   2.0125   2.4360   2.5743   2.6043   2.6262   2.8421
     3.3526   3.6455   3.9498   3.9741   4.0431   4.2443   4.3331   4.4295
     4.4386   4.4966   4.6044   5.1925   5.6498   5.7283

          k = 0.0000 0.0000 1.7142     band energies (ev):

   -19.3017 -13.0566 -11.8637  -8.5581  -7.3506   0.2951   1.1728   1.4385
     2.2212   2.2948   2.4188   2.8440   2.9209   2.9633   2.9950   3.2262
     3.7523   4.0529   4.2302   4.3233   4.4494   4.6127   4.7400   4.8403
     4.8480   4.9063   4.9426   5.5480   6.0237   6.0301

          k = 0.0000 0.0000 1.9284     band energies (ev):

   -18.9955 -13.1837 -12.0864  -8.7164  -6.9564  -0.0746   1.4202   1.8717
     2.6377   2.7566   2.8796   3.3055   3.3161   3.3555   3.4572   3.6743
     4.2036   4.5107   4.5752   4.7374   4.9062   5.0378   5.2010   5.3060
     5.3111   5.3577   5.3714   5.9730   6.3971   6.4108

          k = 0.0000 0.0000 2.1427     band energies (ev):

   -18.6563 -13.6156 -11.9779  -8.9125  -6.5212  -0.5610   1.6755   2.3527
     3.1128   3.2722   3.3943   3.7790   3.8083   3.8258   3.9746   4.1812
     4.7063   4.9852   5.0439   5.1962   5.4033   5.5273   5.7156   5.8240
     5.8264   5.8447   5.8914   6.4588   6.5515   6.8454

          k = 0.0000 0.0000 2.3570     band energies (ev):

   -18.2858 -14.1575 -11.7265  -9.1482  -6.0480  -1.1253   1.9542   2.8798
     3.6487   3.8415   3.9623   4.3088   4.3355   4.3927   4.5464   4.7434
     5.2598   5.4951   5.6103   5.6891   5.8042   6.0801   6.2381   6.2834
     6.3941   6.4015   6.4418   6.4664   7.0047   7.3620

          k = 0.0000 0.0000 2.5713     band energies (ev):

   -17.8878 -14.7165 -11.4224  -9.4192  -5.5415  -1.7303   2.2735   3.4507
     4.2450   4.4640   4.5821   4.9026   4.9238   5.0122   5.1722   5.3553
     5.3663   5.8652   6.0723   6.2339   6.2594   6.6823   6.6991   6.9030
     6.9767   7.0314   7.0429   7.0963   7.5686   7.8240

          k = 0.0000 0.0000 2.7855     band energies (ev):

   -17.4703 -15.2569 -11.0938  -9.7191  -5.0143  -2.3428   2.6453   4.0558
     4.7419   4.9099   5.1389   5.2485   5.5570   5.5815   5.6948   5.8514
     6.0081   6.5156   6.6595   6.7105   6.9111   7.2248   7.3288   7.3714
     7.5689   7.7157   7.7209   7.7574   7.7807   8.0646

          k = 0.0000 0.0000 2.9998     band energies (ev):

   -17.0631 -15.7408 -10.7585 -10.0362  -4.5019  -2.9118   3.0728   4.1302
     4.7340   5.6131   5.8618   5.8642   6.2092   6.2669   6.2936   6.4192
     6.5821   6.8066   7.1442   7.2517   7.3248   7.5685   7.6341   7.6405
     7.6987   7.8629   7.8678   8.0565   8.0966   8.1184

          k = 0.0000 0.0000-3.2141     band energies (ev):

   -16.8305 -15.9998 -10.5075 -10.2834  -4.2084  -3.2260   3.4658   3.6895
     5.3584   5.4248   6.3511   6.4096   6.6168   6.6320   6.7634   6.8018
     7.0324   7.0382   7.0662   7.0767   7.2118   7.2182   7.3707   7.3872
     7.5392   7.5712   7.9752   7.9996   8.1564   8.2117

          k = 0.0000 0.0000-2.9998     band energies (ev):

   -17.0631 -15.7408 -10.7585 -10.0362  -4.5019  -2.9118   3.0728   4.1302
     4.7340   5.6131   5.8618   5.8642   6.2092   6.2669   6.2936   6.4192
     6.5821   6.8066   7.1442   7.2517   7.3248   7.5685   7.6341   7.6405
     7.6987   7.8629   7.8678   8.0565   8.0966   8.1184

          k = 0.0000 0.0000-2.7855     band energies (ev):

   -17.4703 -15.2569 -11.0938  -9.7191  -5.0143  -2.3428   2.6453   4.0558
     4.7419   4.9099   5.1389   5.2485   5.5570   5.5815   5.6948   5.8514
     6.0081   6.5156   6.6595   6.7105   6.9111   7.2248   7.3288   7.3714
     7.5689   7.7157   7.7209   7.7574   7.7807   8.0646

          k = 0.0000 0.0000-2.5713     band energies (ev):

   -17.8878 -14.7165 -11.4224  -9.4192  -5.5415  -1.7303   2.2735   3.4507
     4.2450   4.4640   4.5821   4.9026   4.9238   5.0122   5.1722   5.3553
     5.3663   5.8652   6.0723   6.2339   6.2594   6.6823   6.6991   6.9030
     6.9767   7.0314   7.0429   7.0963   7.5686   7.8240

          k = 0.0000 0.0000-2.3570     band energies (ev):

   -18.2858 -14.1575 -11.7265  -9.1482  -6.0480  -1.1253   1.9542   2.8798
     3.6487   3.8415   3.9623   4.3088   4.3355   4.3927   4.5464   4.7434
     5.2598   5.4951   5.6103   5.6891   5.8042   6.0801   6.2381   6.2834
     6.3941   6.4015   6.4418   6.4664   7.0047   7.3620

          k = 0.0000 0.0000-2.1427     band energies (ev):

   -18.6563 -13.6156 -11.9779  -8.9125  -6.5212  -0.5610   1.6755   2.3527
     3.1128   3.2722   3.3943   3.7790   3.8083   3.8258   3.9746   4.1812
     4.7063   4.9852   5.0439   5.1962   5.4033   5.5273   5.7156   5.8240
     5.8264   5.8447   5.8914   6.4588   6.5515   6.8454

          k = 0.0000 0.0000-1.9284     band energies (ev):

   -18.9955 -13.1837 -12.0864  -8.7164  -6.9564  -0.0746   1.4202   1.8717
     2.6377   2.7566   2.8796   3.3055   3.3161   3.3555   3.4572   3.6743
     4.2036   4.5107   4.5752   4.7374   4.9062   5.0378   5.2010   5.3060
     5.3111   5.3577   5.3714   5.9730   6.3971   6.4108

          k = 0.0000 0.0000-1.7142     band energies (ev):

   -19.3017 -13.0566 -11.8637  -8.5581  -7.3506   0.2951   1.1728   1.4385
     2.2212   2.2948   2.4188   2.8440   2.9209   2.9633   2.9950   3.2262
     3.7523   4.0529   4.2302   4.3233   4.4494   4.6127   4.7400   4.8403
     4.8480   4.9063   4.9426   5.5480   6.0237   6.0301

          k = 0.0000 0.0000-1.4999     band energies (ev):

   -19.5737 -13.1679 -11.3839  -8.4347  -7.7017   0.5202   0.9257   1.0549
     1.8578   1.8871   2.0125   2.4360   2.5743   2.6043   2.6262   2.8421
     3.3526   3.6455   3.9498   3.9741   4.0431   4.2443   4.3331   4.4295
     4.4386   4.4966   4.6044   5.1925   5.6498   5.7283

          k = 0.0000 0.0000-1.2856     band energies (ev):

   -19.8108 -13.3361 -10.8381  -8.3407  -8.0091   0.6032   0.6786   0.7264
     1.5336   1.5426   1.6610   2.0820   2.2267   2.3253   2.3273   2.5276
     3.0046   3.2917   3.6183   3.6893   3.7854   3.9225   3.9807   4.0734
     4.0835   4.1426   4.3336   4.9126   5.3063   5.5053

          k = 0.0000 0.0000-1.0714     band energies (ev):

   -20.0121 -13.5004 -10.3024  -8.2696  -8.2653   0.4203   0.4765   0.5846
     1.2343   1.2728   1.3644   1.7821   1.9274   2.0604   2.0978   2.2791
     2.7083   2.9922   3.3327   3.3889   3.6317   3.6449   3.6865   3.7721
     3.7828   3.8461   4.1092   4.7067   5.0080   5.3382

          k = 0.0000 0.0000-0.8571     band energies (ev):

   -20.1774 -13.6432  -9.8187  -8.4840  -8.1998   0.1876   0.2793   0.5130
     0.9894   1.0487   1.1228   1.5363   1.6821   1.8325   1.9098   2.0844
     2.4637   2.7470   3.0962   3.1421   3.4140   3.4315   3.5058   3.5260
     3.5366   3.6135   3.9188   4.5576   4.7584   5.1566

          k = 0.0000 0.0000-0.6428     band energies (ev):

   -20.3062 -13.7586  -9.4183  -8.6512  -8.1309   0.0029   0.1216   0.4275
     0.7988   0.8715   0.9361   1.3447   1.4912   1.6511   1.7601   1.9352
     2.2710   2.5562   2.9110   2.9491   3.2324   3.2432   3.3336   3.3450
     3.3642   3.4584   3.7637   4.4359   4.5680   4.9660

          k = 0.0000 0.0000-0.4285     band energies (ev):

   -20.3985 -13.8426  -9.1347  -8.7716  -8.0565  -0.1309   0.0062   0.3535
     0.6627   0.7434   0.8034   1.2074   1.3548   1.5200   1.6510   1.8290
     2.1311   2.4199   2.7782   2.8103   3.1019   3.1077   3.1968   3.2080
     3.2409   3.3643   3.6491   4.3119   4.4561   4.8295

          k = 0.0000 0.0000-0.2143     band energies (ev):

   -20.4540 -13.8937  -8.9732  -8.8441  -7.9902  -0.2120  -0.0640   0.3047
     0.5809   0.6658   0.7241   1.1249   1.2729   1.4408   1.5845   1.7654
     2.0457   2.3381   2.6983   2.7265   3.0234   3.0262   3.1146   3.1257
     3.1628   3.3094   3.5791   4.2099   4.4045   4.7476

     highest occupied, lowest unoccupied level (ev):    -4.2084   -3.2260

     Writing output data file poly_pa.save
 
     init_run     :      0.22s CPU      0.23s WALL (       1 calls)
     electrons    :    174.00s CPU    176.20s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.00s CPU      0.00s WALL (       1 calls)
     potinit      :      0.05s CPU      0.04s WALL (       1 calls)

     Called by electrons:
     c_bands      :    174.00s CPU    176.18s WALL (       1 calls)
     v_of_rho     :      0.02s CPU      0.01s WALL (       1 calls)

     Called by c_bands:
     init_us_2    :      0.01s CPU      0.05s WALL (      30 calls)
     cegterg      :    164.43s CPU    165.52s WALL (      71 calls)

     Called by *egterg:
     h_psi        :    135.59s CPU    136.08s WALL (    1280 calls)
     g_psi        :      1.55s CPU      1.49s WALL (    1179 calls)
     cdiaghg      :      6.81s CPU      6.73s WALL (    1209 calls)

     Called by h_psi:
     add_vuspsi   :      2.09s CPU      1.91s WALL (    1280 calls)

     General routines
     calbec       :      2.40s CPU      2.40s WALL (    1280 calls)
     fft          :      0.02s CPU      0.02s WALL (       3 calls)
     fftw         :    121.31s CPU    122.03s WALL (   41152 calls)
     davcio       :      0.12s CPU      1.00s WALL (      60 calls)
 
     Parallel routines
     fft_scatter  :     30.68s CPU     30.39s WALL (   41155 calls)
 
     PWSCF        :  2m56.29s CPU     2m59.64s WALL

 
   This run was terminated on:  20:40:34  17Apr2014            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
